src/hyperion/__main__.py

```diff
@@ -221,6 +221,9 @@ def get(self, **kwargs):
         action = kwargs.get("action")
         status_and_message = StatusAndMessage(Status.FAILED, f"{action} not understood")
         if action == Actions.STATUS.value:
+            hyperion.log.LOGGER.debug(
+                f"Runner recieved status request - state of the runner object is: {self.runner.__dict__} - state of the RE is: {self.runner.RE.__dict__}"
+            )
```

Comment:
Could: Not sure how useful this is in production code. I would remove it but if you think it's useful that's fine

Reply:
I thought it might actually be handy some time to be able to make it dump the RE status